client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -475,8 +475,8 @@ export class PluginsBrowser extends Component {
 		return (
 			<HeaderButton
 				icon="cloud-upload"
-				label={ translate( 'Upload Plugin' ) }
-				aria-label={ translate( 'Upload Plugin' ) }
+				label={ translate( 'Install Plugin' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Install plugin' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/install-action-messaging)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).